andela/troupon
# Bg load modals on internal pages 121394477

### What Does This PR Do?
This PR makes modals for cities,merchants and categories to load on internal pages for user's profile. 

### Description Of Task To Be Completed
* On the profile pages, the cities, merchants and categories modal should contain the correct information.

### How Should This Be Tested Manually?
Go to User profile and click categories, cities and/or merchants tabs. Appropriate data should be displayed respectively

### What Are The Relevant Pivotal Tracker Stories?
#121394477

### ScreenShots
<img width="1279" alt="screen shot 2016-06-27 at 4 46 22 pm" src="https://cloud.githubusercontent.com/assets/17295618/16381935/6bfd4abe-3c87-11e6-93c3-54e0c9dec357.png">
<img width="1280" alt="screen shot 2016-06-27 at 4 52 51 pm" src="https://cloud.githubusercontent.com/assets/17295618/16382012/bf9115f2-3c87-11e6-8068-e134e82e0977.png">
<img width="1280" alt="screen shot 2016-06-27 at 4 53 25 pm" src="https://cloud.githubusercontent.com/assets/17295618/16382011/bf8f7b3e-3c87-11e6-9538-eeeb65ec31cd.png">
